state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -1054,6 +1075,34 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		#[pallet::call_index(12)]
+		#[pallet::weight(T::WeightInfo::open_private_channel())]
+		pub fn open_private_channel(origin: OriginFor<T>, chain: ForeignChain) -> DispatchResult {
+			let broker_id = T::AccountRoleRegistry::ensure_broker(origin)?;
+
+			ensure!(chain == ForeignChain::Bitcoin, Error::<T>::NoPrivateChannelsForChain);
```

Comment:
Now we only inject the BitcoinIngressEgress as the PrivateChannelManager, so it's only possible to create bitcoin channel, or inversely, if we injected the wrong instance of the ingress egress then we would put bitcoin in here, and then open an Ethereum channel. 
 So perhaps would be better to have this be:
```fn open_private_btc_channel(origin)``` and we can remove this ensure, same below.

Reply:
Was going to suggest the same thing - the only benefit of the `chain` parameter is in case we want to add another utxo chain... but that seems unlikely right now and in the meantime it's not used for anything.